dash/include/dash/algorithm/Copy.h

```diff
@@ -88,12 +93,14 @@ OutputIt copy(
  * \ingroup  DashAlgorithms
  */
 template <
-  typename ValueType,
+  class ExecutionPolicy,
+  class ValueType,
   class GlobInputIt >
-dash::Future<ValueType *> copy_async(
-  InputIt  in_first,
-  InputIt  in_last,
-  OutputIt out_first);
+dash::Future<ValueType *> copy(
+  ExecutionPolicy && policy,
```

Comment:
Yes, I just pinned the branch as PR to simplify discussion. It's work in progress (= `[WIP]` in title).